packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/CryptoPayment.tsx

```diff
@@ -363,7 +367,7 @@ const CryptoPayment = ({ goBack, planPrice }: ICryptoPayment) => {
           />
         </div>}
       </div>
-      {cryptoChargeLoading && <div className={classes.loadingContainer}>
+      {(cryptoChargeLoading || (pendingCryptoInvoice && !timeRemaining)) && <div className={classes.loadingContainer}>
```

Comment:
would this be better ? 
`cryptoChargeLoading || (pendingCryptoInvoice && !cryptoPayment))` ?

I'm just worried timeRemaining goes to 0, and loading comes back

Reply:
Sure, that should be the same though, because there shouldn't be any pendinCryptoIncoive once there's no more time, but we can try this out.
```suggestion
      {(cryptoChargeLoading || (pendingCryptoInvoice && !cryptoPayment)) && <div className={classes.loadingContainer}>
```

Reply:
integrated